docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+| R6   | Unable to control payments                                 | critical | <ul><li>Ability to control when, who & how much to pay</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+<!-- prettier-ignore-end -->
+
+#### Basic Test Cases
+
+##### Connect to a wallet
+
+**Test ID**: 1  
+**Description**: Connect the extension to a Web Monetization enabled digital wallet
+**Risk**: R2 (onboarding) and R5 (UX)  
+**Preconditions**:
+
+1. You have already downloaded and installed the Web Monetization extension.
+2. Your digital wallet sign-up and account activation is complete (learn more from the [Prerequisites](#Prerequisites) section)
+
+**Steps**:
+
+1. Open the WM extension and copy the extension’s key.
+2. Open the digital wallet, and load the extension’s key into the wallet.
+3. From the digital wallet, find and copy the wallet address or payment pointer.
+4. Open the WM extension, and enter the wallet address or payment pointer.
+5. Enter the amount you want to make available to the extension from your wallet.
+6. Keep the option to renew monthly disabled.
+
+**Expected results**:
+
+1. The wallet owner receives an interaction prompt from their wallet to authorize the connection and access to the amount.
+2. On accepting, you get shown the message "Your wallet is now successfully connected to the extension."
+
+##### Make continuous payments
+
+**Test ID**: 2  
+**Description**: Send continuous WM payments to a monetized website  
+**Risk**: R3 (security of funds)  
+**Preconditions**:
+
+1. The extension is connected to your wallet.
+2. The extension has a positive remaining balance.
+3. Your wallet balance is equal to or greater than the extension’s remaining balance.
+
+**Steps**:
+
+1. Visit a monetized website. Refer to the [Test Data](#Test-Data) section to explore different WM conditions for websites.
```

Comment:
I mentioned earlier that I didn't know how useful/helpful it was to have that Test Data section. I'm commenting here in case that section comes out. 

The less familiar a tester is with WM, the more I'm afraid that the test playground is going to be totally meaningless and confusing to them. Ideally, we want to mimic the conditions under which WM payments would be made. Which means the tester would simply go to a site. They wouldn't have to go to a test site and plug in 1+ wallet addresses. Just my 2 cents.

Reply:
Thanks Melissa. 
Please have a look at the revised **Test Data** section.
It's more explicit, and hopefully clearer about (1) websites that can be used during testing and (2) wallet addresses that a tester can use on the WM playground.